src/column/writer.rs

```diff
@@ -298,9 +298,7 @@ impl<T: DataType> ColumnWriterImpl<T> {
       let levels = def_levels.unwrap();
       num_values = levels.len();
       for &level in levels {
-        if level == self.descr.max_def_level() {
-          values_to_write += 1;
-        }
+        values_to_write += (level == self.descr.max_def_level()) as usize & 1;
```

Comment:
@sunchao I just realised, should we remove `& 1`, since it is not really needed - boolean is either 0 or 1. I can open PR to fix it.

Reply:
Yes we should not need that `&`. Feel free to add a PR to fix it. :)